apps/roam/src/components/settings/Settings.tsx

```diff
@@ -100,6 +100,7 @@ export const SettingsDialog = ({
         <style>{`
           .dg-settings-tabs .bp3-tab-list {
             overflow-y: auto;
+            overflow-x: hidden;
```

Comment:
This works, but seeing as none of the text is long enough to cause the issue, this might be more of a bandaid.  And might bite us later.  I won't block on this, but I wonder what the root cause actually is?